tests/unit/_internal/pip_api/test_install_requirements.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import pytest
+
+from piptools._internal import _pip_api
+
+
+@pytest.mark.skipif(
+    _pip_api.PIP_VERSION_MAJOR_MINOR < (25, 3), reason="test requires pip>=25.3"
+)
```

Comment:
Marks used multiple times can be defined once and reused as follows:
```suggestion
requires_pip_25_3 = pytest.mark.skipif(
    _pip_api.PIP_VERSION_MAJOR_MINOR < (25, 3),
    reason="test requires pip>=25.3",
)


@requires_pip_25_3
```

Reply:
Ah, I missed that. Marking as resolved.